requirements.txt

```diff
@@ -1 +1 @@
-commisery>=0.9.1,<1
\ No newline at end of file
+git+https://github.com/tomtom-international/commisery.git@dcd4094e199b24e06843b8ed152209e29711a97d
```

Comment:
Note that this shall be reset to `commisery>=1,<2` as soon as Commisery v1.0.0 is released.